### PR TITLE
feat: Refactor alias topdown and versioning action menus

### PR DESCRIPTION
**Pull request by NarenderRajuB**

### CODE REVIEW

File: djangocms_alias/admin.py [hunk 1]
```diff
@@ -171,12 +171,12 @@ def _get_alias_usage_link(self, obj, request):
...
         )
 
-    def _get_alias_site_category_change_link(self, obj, request):
+    def _get_change_alias_settings_link(self, obj, request):
         url = admin_reverse('{}_{}_change'.format(
             obj._meta.app_label, obj.alias._meta.model_name), args=(obj.alias.pk,)
         )
         return render_to_string(
-            "admin/djangocms_alias/icons/edit_alias.html",
+            "admin/djangocms_alias/icons/change_alias_settings.html",
```
Comment (adam-murray) [inline djangocms_alias/admin.py:179]: Could we have disabled=False as a param coming into this function? That way if someone needs it to be disabled, they won't have to override the entire method.

**Review by Bernardvdv — APPROVED** [review]

File: djangocms_alias/cms_toolbars.py [hunk 2]
```diff
@@ -75,47 +76,49 @@ def add_alias_menu(self):
...
+                USAGE_ALIAS_URL_NAME,
+                args=[self.toolbar.obj.alias_id],
             ),
-            disabled=disabled,
         )
 
+        # Only show deletion if versioning is not enabled
+        if not is_versioning_enabled():
+            alias_menu.add_modal_item(
+                _('Delete Alias'),
```
Comment (adam-murray) [inline djangocms_alias/cms_toolbars.py:110]: The capitalisation here isn't consistent with the versioned menu option.

**Review by adam-murray — CHANGES_REQUESTED** [review]
Looks good, only a small inconsistency in menu capitalisation.

File: djangocms_alias/cms_toolbars.py (hunk 2 above)
Comment (Aiky30) [reply djangocms_alias/cms_toolbars.py:110]: This is where suggestions come into their own. 

```suggestion
                _('Delete alias'),
```

**Review by adam-murray — APPROVED** [review]

**Pull request merged**